packages/medusa-telemetry/src/telemeter.js

```diff
@@ -82,7 +82,10 @@ class Telemeter {
 
   getMedusaVersion() {
     try {
-      const packageJson = require.resolve(`@medusajs/medusa/package.json`)
+      const packageJson = join(
+        require.resolve(`@medusajs/medusa`).split(`${sep}dist`).shift(),
```

Comment:
Can we instead expose the package.json file from the package exports?

Reply:
yes we can, let me do that